Jellyfin.Server/Migrations/Routines/RemoveDuplicateExtras.cs

```diff
@@ -37,14 +38,12 @@ public void Perform()
         {
             var dataPath = _paths.DataPath;
             var dbPath = Path.Combine(dataPath, DbFilename);
-            using (var connection = SQLite3.Open(
-                dbPath,
-                ConnectionFlags.ReadWrite,
-                null))
+            using (var connection = new SqliteConnection($"Filename={dbPath}"))
             {
                 // Query the database for the ids of duplicate extras
                 var queryResult = connection.Query("SELECT t1.Path FROM TypedBaseItems AS t1, TypedBaseItems AS t2 WHERE t1.Path=t2.Path AND t1.Type!=t2.Type AND t1.Type='MediaBrowser.Controller.Entities.Video'");
-                var bads = string.Join(", ", queryResult.SelectScalarString());
+                // TODO does this LINQ execute before the reader is disposed?
+                var bads = string.Join(", ", queryResult.Select(x => x.GetString(0)).ToList());
```

Comment:
Shouldn't even need the tolist here, no?

Reply:
No